modules/apps/jaggery/mdm/src/mdm/units/platform-configuration/platform-configuration.hbs

```diff
@@ -18,7 +18,7 @@
                     {{/each}}
                     ">
                         <div class="wr-hidden-operations-nav col-lg-4">
-                            <a href="javascript:void(0)" onclick="showAdvanceOperation('general', this)" class="selected">
+                            <a id=generalConfigLink href="javascript:void(0)" onclick="showAdvanceOperation('general', this)" class="selected">
```

Comment:
unquoted attributes are valid on html5 specification, but as a best practice do recommend to use quotes.